build_tools/update_shape_lib.sh

```diff
@@ -1,13 +1,22 @@
 #!/bin/bash
 # Updates auto-generated shape library files for the `torch` dialect.
```

Comment:
maybe add a "usage:" in the comment at the top of this file covering the new env vars.